app/src/main/java/com/framgia/fsalon/utils/binding/BindingUtils.java

```diff
@@ -10,6 +10,7 @@
 import android.support.v4.view.ViewPager;
 import android.support.v7.widget.RecyclerView;
 import android.view.View;
+import android.view.ViewGroup;
```

Comment:
ERROR: Unused import: android.view.ViewGroup. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports